src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -296,14 +296,14 @@ public static CodeAction Create(string title, Func<CancellationToken, Task<Solut
         /// <summary>
         /// Creates a top level code action with multiple code action choices a user can invoke.
         /// </summary>
-        public static CodeAction Create(string title, IEnumerable<CodeAction> nestedActions, string equivalenceKey = null)
+        public static CodeAction Create(string title, IEnumerable<CodeAction> nestedActions)
         {
             if (title == null)
             {
                 throw new ArgumentNullException(nameof(title));
             }
 
-            return new SimpleCodeAction(title, nestedActions.AsImmutableOrEmpty(), equivalenceKey);
+            return new SimpleCodeAction(title, nestedActions.AsImmutableOrEmpty(), equivalenceKey: string.Empty);
```

Comment:
Additionally, can you please improve upon the test that I added for [this](https://github.com/dotnet/roslyn/issues/3311) bug:

1. Extend [this](http://source.roslyn.io/#Roslyn.Services.Editor.CSharp.UnitTests/Diagnostics/Suppression/SuppressionTests.cs,200) test to verify `Assert.NotEqual(string.Empty, fixes.First().Action.EquivalenceKey)`

2. Clone or extend the same test to verify that if the same line had diagnostics with different IDs, then the suppression fixes for them have different equivalence keys.